### PR TITLE
(test) O3-2218 O3-2219: Add test for `add-patient-link` and `nav-link tests` components

### PR DESCRIPTION
**Pull request by ayush-AI**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

In this PR I have added tests for add-patient-link and nav-link components.

## Screenshots

*None.*

## Related Issue

https://issues.openmrs.org/browse/O3-2218
https://issues.openmrs.org/browse/O3-2219

## Other

*None.*

### CODE REVIEW

**Comment by ayush-AI** [comment]
cc @jayasanka-sack @anjula-sack @Piumal1999

File: packages/esm-patient-registration-app/src/add-patient-link.test.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import * as esmFramework from '@openmrs/esm-framework';
+import Root from './add-patient-link';
+
+describe('Root component', () => {
```
Comment (anjula-sack) [inline packages/esm-patient-registration-app/src/add-patient-link.test.tsx:6]:
```suggestion
describe('Add patient button', () => {
```

File: packages/esm-patient-registration-app/src/nav-link.test.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Root from './nav-link';
+
+describe('Root component', () => {
```
Comment (anjula-sack) [inline packages/esm-patient-registration-app/src/nav-link.test.tsx:5]:
```suggestion
describe('Nav link component', () => {
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — APPROVED** [review]
Good job! @ayush-AI

**Pull request merged**